examples/http-service-basic-authentication-ldap-user-store/http_service_basic_authentication_ldap_user_store.md

```diff
@@ -1,11 +1,11 @@
 # HTTP service - Basic authentication LDAP user store
 
-An HTTP service/resource can be secured with basic authentication and by enforcing authorization optionally. Then, it validates the Basic Auth token sent in the `Authorization` header against the provided configurations. This reads data from the configured LDAP. This stores usernames, passwords for authentication, and scopes for authorization. Ballerina uses the concept of scopes for authorization. A resource declared in a service can be bound to one/more scope(s). In the authorization phase, the scopes of the service/resource are compared against the scope included in the user store for at least one match between the two sets.
+An `http:Service` can be secured with basic authentication and optionally by enforcing authorization. This can be achieved by validating the basic authentication token sent in the `Authorization` header with the LDAP server. This server stores the usernames and passwords for the authentication and the scopes for the authorization. An `http:Service` can configure the scopes it needs for authorization in the `auth` field of the `@http:ServiceConfig` annotation. A `401 Unauthorized` response is sent to the client when the authentication fails, and a `403 Forbidden` response is sent to the client when the authorization fails. Use this to authenticate and authorize requests based on LDAP user stores. Furthermore, the authentication and authorization configurations can be overwritten at the resource level using the `@http:ResourceConfig` annotation.
```

Comment:
```suggestion
The `http:Service` can be secured with basic authentication and optionally by enforcing authorization. This can be achieved by validating the basic authentication token sent in the `Authorization` header with the LDAP server. This server stores the usernames and passwords for the authentication and the scopes for the authorization. An `http:Service` can configure the scopes it needs for authorization in the `auth` field of the `@http:ServiceConfig` annotation. A `401 Unauthorized` response is sent to the client when the authentication fails, and a `403 Forbidden` response is sent to the client when the authorization fails. Use this to authenticate and authorize requests based on LDAP user stores. Furthermore, the authentication and authorization configurations can be overwritten at the resource level using the `@http:ResourceConfig` annotation.
```